app/assets/stylesheets/application.scss

```diff
@@ -10,6 +10,9 @@
  * files in this directory. Styles in this file should be added after the last require_* statement.
  * It is generally better to create a new file per style scope.
  *
+ *= require datatables.net-bs/css/dataTables.bootstrap.css
+ *= require datatables.net-fixedheader-bs/css/fixedHeader.bootstrap.css
+ *= require datatables.net-dt/css/jquery.dataTables.css
```

Comment:
Datatables docs say don't do both.

>  For CSS only a single file should be included as the styling library will provide the core styling options for the table. The CSS files have the format dataTables.{style}.css (dataTables.bootstrap.css for Bootstrap for example).

As a hint:  Try adding the class `.table` to the table to get it to be a [Bootstrap table](https://getbootstrap.com/docs/3.3/css/#tables)

>  For basic styling — light padding and only horizontal dividers — add the base class `.table` to any `<table>`. It may seem super redundant, but given the widespread use of tables for other plugins like calendars and date pickers, we've opted to isolate our custom table styles.

Reply:
Sure but that takes away the arrows.

Reply:
I guess I can leave the original arrows and it won't close #60 and that's fine.

Reply:
I am wrong! I removed the wrong one. We're good.